utils/wget.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<interface uri="http://repo.roscidus.com/utils/wget" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd" xmlns="http://zero-install.sourceforge.net/2004/injector/interface">
+  <name>GNU Wget</name>
+  <summary xml:lang="en">static builds of Wget with taskbar progressbar Windows certificate store support, manual</summary>
+  <description xml:lang="en">GNU Wget is a free software package for retrieving files using HTTP, HTTPS, FTP and FTPS the most widely-used Internet protocols. It is a non-interactive commandline tool, so it may easily be called from scripts, cron jobs, terminals without X-Windows support, etc. 
+
+GNU Wget has many features to make retrieving large files or mirroring entire web or FTP sites easy, including: 
+◾Can resume aborted downloads, using REST and RANGE
+◾Can use filename wild cards and recursively mirror directories
+◾NLS-based message files for many different languages
+◾ Optionally converts absolute links in downloaded documents to relative, so that downloaded documents may link to each other locally 
+◾Runs on most UNIX-like operating systems as well as Microsoft Windows
+◾Supports HTTP proxies
+◾Supports HTTP cookies
+◾Supports persistent HTTP connections
+◾Unattended / background operation
+◾Uses local file timestamps to determine whether documents need to be re-downloaded when mirroring
+◾GNU Wget is distributed under the GNU General Public License.
+</description>
+  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.ico" type="image/vnd.microsoft.icon" />
+  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.png" type="image/png" />
+  <category>Network</category>
+  <homepage>https://eternallybored.org/misc/wget/</homepage>
+  <needs-terminal />
+  <group arch="Windows-*" license="GPL v3 (GNU General Public License)">
```

Comment:
We can leave out `arch="Windows-*"` here, since the arch gets overriden by all implementations in the group.

Reply:
done